classes/profile_table.php

```diff
@@ -97,4 +127,18 @@ public function col_created(object $record): string {
     public function col_parameters(object $record): string {
         return htmlentities($record->parameters);
     }
+
+    /**
+     * Displays the full name of the user.
+     *
+     * @param object $record
+     * @return string
+     */
+    public function col_user(object $record): string {
+        if ($record->userid == 0) {
+            return '-';
+        } else {
+            return fullname($record);
```

Comment:
can you please link this to the user profle

auth/catadmin/classes/form/admin_report.php:131:            $fullname = \html_writer::link('/user/profile.php?id=' . $user->id, $fullname);

Reply:
done in both the table and the template.